patches/tModLoader/Terraria/Main.cs.patch

```diff
@@ -2648,6 +2648,143 @@
  						if ((!expertMode || type != 266) && ((type != 439 && type != 440) || npc[num2].ai[0] != 5f)) {
  							if (type >= 134 && type <= 136) {
  								scale = 1.5f;
+@@ -34675,6 +_,7 @@
+ 			if (!CanShowInfoAccs)
+ 				return;
+ 
++			/*
```

Comment:
These were the multilines I was talking about.

Reply:
These actually probably aren't necessary; I can get rid of 'em if desired.